src/boundary/Obstacle.h

```diff
@@ -20,40 +20,41 @@
 class Obstacle {
  public:
     Obstacle(real x1, real x2, real y1, real y2, real z1, real z2, const std::string& name);
-    Obstacle(size_t coords_i1, size_t coords_j1, size_t coords_k1, size_t coords_i2, size_t coords_j2, size_t coords_k2, size_t level, const std::string& name);
+    Obstacle(size_t coords_i1, size_t coords_j1, size_t coords_k1, size_t coords_i2,
+             size_t coords_j2, size_t coords_k2, size_t level, const std::string& name);
     ~Obstacle();
 
-    size_t* getObstacleList() { return m_obstacleList; }
-    size_t* getObstacleFront() { return m_obstacleFront; }
-    size_t* getObstacleBack() { return m_obstacleBack; }
-    size_t* getObstacleTop() { return m_obstacleTop; }
-    size_t* getObstacleBottom() { return m_obstacleBottom; }
-    size_t* getObstacleLeft() { return m_obstacleLeft; }
-    size_t* getObstacleRight() { return m_obstacleRight; }
+    size_t* get_obstacle_list() { return m_obstacle_list; }
+    size_t* get_obstacle_front() { return m_obstacle_front; }
+    size_t* get_obstacle_back() { return m_obstacle_back; }
+    size_t* get_obstacle_top() { return m_obstacle_top; }
+    size_t* get_obstacle_bottom() { return m_obstacle_bottom; }
+    size_t* get_obstacle_left() { return m_obstacle_left; }
+    size_t* get_obstacle_right() { return m_obstacle_right; }
```

Comment:
could be const

Reply:
85bf1063a3b2f3a9b442eb00a8da84c334d4611a